### PR TITLE
Publish build symbols

### PR DESCRIPTION
**Pull request by tiagoshibata**
**Description**: Publish debug symbols to VSTS symbol server

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml [hunk 1]
```diff
@@ -72,6 +72,13 @@ jobs:
       workingFolder: '$(Build.BinariesDirectory)\$(BuildConfig)'
       createLogFile: true
 
+  - task: PublishSymbols@2
+    displayName: 'Publish Build Symbols'
+    inputs:
+      symbolsFolder: $(Build.BinariesDirectory)\$(BuildConfig)
+      searchPattern: '**/*.pdb'
```
Comment (snnn) [inline tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml]: Probably you only need the onnxruntime.pdb.

Comment (tiagoshibata) [reply tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml]: Also WinML/DML, and possibly more stuff that might get built or added in the future. I thought it was better to upload everything.

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml]: PDBs are huge. We may run out of storage quickly.

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml]: "symbolsFolder: $(Build.BinariesDirectory)\$(BuildConfig)\$(BuildConfig)" would be better.

**Pull request merged**